akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -512,6 +515,14 @@ final case class SuppressedDeadLetter(message: DeadLetterSuppression, sender: Ac
   require(recipient ne null, "DeadLetter recipient may not be null")
 }
 
+/**
+ * Envelope that is published on the eventStream wrapped in [[akka.actor.DeadLetter]] for every message that is
+ * dropped due to overfull queues or routers with no routees.
+ */
+final case class Dropped(message: Any, reason: String, recipient: ActorRef) extends AllDeadLetters {
```

Comment:
Do we have a clear-cut definition about for which things dropped should be used and when DeadLetter? Or now that we have a precedence, add more subclasses to `AllDeadLetters`?

Reply:
`Dropped` is documented as:
```
/**
 * Envelope that is published on the eventStream for every message that is
 * dropped due to overfull queues or routers with no routees.
 */
```

while "ordinary" `DeadLetter` is when the destination actor has been terminated or similar.

another place where could use `Dropped` is when a bounded mailbox drops.

Reply:
:+1: thanks for clarification